ci/driver.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash --login
+
+my_dir="$( cd "$( dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd )"
+
+# ==============================================================================
+usage() {
+  set +x
+  echo
+  echo "Usage: $0 -t <target> -h"
+  echo
+  echo "  -t  target/machine script is running on    DEFAULT: $(hostname)"
+  echo "  -h  display this message and quit"
+  echo
+  exit 1
+}
+
+# ==============================================================================
+# First, set up runtime environment
+
+export TARGET="$(hostname)"
+
+while getopts "t:h" opt; do
+  case $opt in
+    t)
+      TARGET=$OPTARG
+      ;;
+    h|\?|:)
+      usage
+      ;;
+  esac
+done
+
+case ${TARGET} in
+  hera | orion)
+    echo "Running Automated Testing on $TARGET"
+    source $MODULESHOME/init/sh
+    source $my_dir/${TARGET}.sh
+    module purge
+    module use $RDAS_MODULE_USE
+    module load RDAS/$TARGET
+    module list
+    ;;
+  *)
+    echo "Unsupported platform. Exiting with error."
+    exit 1
+    ;;
+esac
+
+# ==============================================================================
+# pull on the repo and get list of open PRs
+cd $RDAS_CI_ROOT/repo
+CI_LABEL="${RDAS_CI_HOST}-RT"
+gh pr list --label "$CI_LABEL" --state "open" | awk '{print $1;}' > $RDAS_CI_ROOT/open_pr_list
+open_pr_list=$(cat $RDAS_CI_ROOT/open_pr_list)
+
+# ==============================================================================
+# clone, checkout, build, test, etc.
+repo_url="https://github.com/NOAA-EMC/RDASApp.git"
+# loop through all open PRs
+for pr in $open_pr_list; do
```

Comment:
Are there any controls to let only those PR with new commits  after last CI test (and successful)) to be tested again?

Reply:
hmm, maybe, just remove the passed label and re-add the queue label is what I've used in the past.